platform/src/main/java/org/hillview/storage/CassandraSSTableLoader.java

```diff
@@ -312,21 +258,25 @@ public ITable load(){
                 IColumnLoader loader = new CassandraSSTableLoader.SSTableColumnLoader(this.ssTableReader, this.actualSchema);
                 return Table.createLazyTable(cds, size, this.filename, loader);
             } else {
-                return nonLazyLoading();
+                int sizeToLoad = this.columns.length;
+                /** columns loader will load all column, so all item of columnToLoad need to be TRUE */
```

Comment:
do not use JavaDoc comments in the code; they are reserved for declarations only.

Reply:
Will do!

Reply:
no need to reply to every comment, just mark them as resolved